include/picongpu/plugins/openPMD/Json.cpp

```diff
@@ -153,38 +153,52 @@ namespace
         return res;
     }
 
-    void readPattern(std::vector<Pattern>& patterns, nlohmann::json const& object)
+    KindOfConfig readPattern(
+        std::vector<Pattern>& patterns,
```

Comment:
I think `#include <vector>` is missing.

Reply:
That one is in `JSON_private.hpp` (as is the declaration of `readPattern`. Should I add it again?

Reply:
Ah, I didn't get this immediately. I think it's fine then.